rebasehelper/application.py

```diff
@@ -396,6 +396,8 @@ def prepare_sources(self):
         old_dir = Application.extract_sources(self.old_sources, old_sources_dir)
         new_dir = Application.extract_sources(self.new_sources, new_sources_dir)
 
+        self.run_package_checkers(self.results_dir, type="SOURCE")
```

Comment:
This should be run after extraction of additional source archives. And it would make more sense to me to run it after patching.